community-website/src/community-project-boilerplate-docgen/src/widgets/panel.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Widgets - Panel
 layout: widget.pug
+canonical: https://www.algolia.com/doc/api-reference/widgets/panel/angular/
```

Comment:
This page does not exist. It looks like we forgot to add it to the main doc.

Reply:
What do you suggest; no canonical link or adding this in the docs first?

Reply:
First add the doc page, then keep the canonical link.

Reply:
Yes, I wasn't sure what to do with the panel yet. 
After our discussion, it seems clear we should almost keep it as is, so I'll do that.

Reply:
https://github.com/algolia/doc/pull/2673